packages/components/src/buttons.test.ts

```diff
@@ -0,0 +1,67 @@
+import { h } from 'react-hyperscript-helpers';
+
+import { ButtonOutline, ButtonPrimary, ButtonSecondary } from './buttons';
+import { Clickable } from './Clickable';
+import { renderWithTheme as render } from './internal/test-utils';
+
+type ClickableExports = typeof import('./Clickable');
+jest.mock('./Clickable', (): ClickableExports => {
+  return {
+    ...jest.requireActual<ClickableExports>('./Clickable'),
+    // @ts-expect-error
```

Comment:
TS is looking for a `$$typeof` field. This seems to be a consequence of `Clickable` forwarding its ref.

Reply:
Added comments in https://github.com/DataBiosphere/terra-ui/pull/4335/commits/928481f3d7dae3c28e40b933466e572cc73a01ea.